src/common/json_helper.h

```diff
@@ -493,6 +494,26 @@ inline bool json_decode(const JsonObject &in, std::set<T> &t)
     return true;
 }
 
+template <typename T1, typename T2>
+inline void json_encode(JsonWriter &out, const std::unordered_set<T1, T2> &t)
```

Comment:
Is it possiable to make std::unordered_set as a template type too? then it can replace the template function above.

Reply:
I don't agree, the template type is too deep, it will only increase the complexity of code reading, not friendly to developers and viewers.

Reply:
Template is often used to reduce duplicate code, now the new added function is duplicate with `inline bool json_decode(const JsonObject &in, std::set<T> &t)`.
Reference to `json_encode_map` how to deal with `map` and `unordered_map`